Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);

```

Comment:
**Code Structure Issue**: The Scanner object 'sc' is created but never used in the Even method, which is unnecessary and can be removed to clean up the code. <br> **Fix**: Remove the unused Scanner object instantiation to clean up the code. <br> **Code Suggestion**: 
 ```
 // Scanner sc = new Scanner(System.in); // Removed unused Scanner object
 ```